src/QMCWaveFunctions/SPOSet.cpp

```diff
@@ -57,6 +57,15 @@ void SPOSet::evaluateDetRatios(const VirtualParticleSet& VP,
   }
 }
 
+void SPOSet::evaluateDetSpinorRatios(const VirtualParticleSet& VP,
+                             ValueVector& psi,
+                             const std::pair<ValueVector, ValueVector>& spinor_multiplier,
+                             const ValueVector& invrow,
+                             std::vector<ValueType>& ratios)
+{
+  throw std::runtime_error("Need specialization of SPOSet::evaluateDetSpinorRatios");
```

Comment:
Please expand the error message with ClassName.

Reply:
fixed